test/e2e/framework/metrics/e2e_metrics.go

```diff
@@ -33,10 +33,10 @@ const (
 	caFunctionMetricLabel = "function"
 )
 
-// MetricsForE2E is metrics collection of components.
-type MetricsForE2E Collection
+// CollectionForE2E is metrics collection of components.
+type CollectionForE2E Collection
```

Comment:
Just throwing ideas out there... how about E2EMetricsCollection ?

Reply:
Actually that stutters worse. Nvm

Reply:
Golint said "Metrics" is redundant because of the package name.
So I don't think it is better to contain "Metircs" in the name.

Reply:
I was thinking that wouldn’t be the case of “e2e” is in between.
I’m not completely sold on “CollectionForE2E” being a good name, sounds vague

Reply:
The package name is already meaning the Collection is a part of "metrics"..

Reply:
I don’t think CollectionForE2E is an improvement over MetricsForE2E in terms of making sense to a consumer of the package. 
Though my opinion is very subjective. 
@andrewsykim wdyt about the new name?

Reply:
Was E2EMetrics ruled out? IIRC, I think golint only complains if Metrics is the first word in the name but it can still contain it.

Reply:
Or `ComponentMetrics` / `ComponentCollection` since this seems to be specific to component level metrics?

Reply:
`ComponentCollection` seems good for me, because I felt `E2E` also seems redundant because  this is clearly a part of e2e framework.
Anyways nice idea, thanks.